packages/carousel/src/react/index.js

```diff
@@ -63,6 +63,7 @@ export default function Carousel({ controls, size, ...props }) {
         {...filterReactProps(props)}
         ref={ref}
       >
+        {controls}
```

Comment:
The ordering change is so that controls come first in tab order, right?  And the associated pointer-event changes so we can still click stuff in the carousel, right?

Reply:
Yep